enterprise/internal/licensing/data.go

```diff
@@ -25,6 +25,9 @@ const (
 
 	// PlanFree1 is the default plan if no license key is set from 4.5 onwards.
 	PlanFree1 Plan = "free-1"
+
+	// PlanAirGappedEnterprise is the same PlanEnterprise1 but with FeatureAllowAirGapped, and works starting from 5.1.
+	PlanAirGappedEnterprise Plan = "air-gapped-enterprise"
```

Comment:
Let's number the plan, since it proved useful to have different versions of plans:
```suggestion
	PlanEnterpriseAirGap0 Plan = "enterprise-air-gap-0"
```

And we also need to modify this list unfortunately: https://sourcegraph.sourcegraph.com/github.com/sourcegraph/sourcegraph/-/blob/client/web/src/jscontext.ts?L254

Reply:
Got it, updated.